app/sessions/session-details/session-details.component.html

```diff
@@ -1,7 +1,7 @@
 <StackLayout class="session-detail-wrapper">
 	<StackLayout *ngIf="session">
 
-		<GridLayout rows="auto, *" (swipe)="backSwipe($event)">
+		<GridLayout rows="auto, auto, *" (swipe)="backSwipe($event)">
```

Comment:
Why was the "auto" added to this layout? I don't see any new changes being made to the html itself. Same below.

Reply:
Auto was added for moving the session titles as its own header (not hover over Pujyashree image) but is now depreciated, so will remove both the auto, * (same for below comment)